subworkflows/local/vcf_annotate_bcftools/main.nf

```diff
@@ -12,18 +12,17 @@ workflow VCF_ANNOTATE_BCFTOOLS {
     annotations_index //
     header_lines      //
 
-
     main:
-    ch_versions = Channel.empty()
+    versions = Channel.empty()
```

Comment:
why?

Reply:
That way I don't have to do:
Most of our "variables" are channels anyway, I think we'd better prefix the other non channel elements.
(But that might be a discussion for another day).

```
emit:
versions = ch_versions
```